### ISSUE TITLE
Page admin: move newly added content object is not executed

### ISSUE THREAD

**Issue by juergmessmer** [rapila/cms-base#248]
Page_detail: A newly added content object (adminAddObjectToContainer) can't be moved directly. "adminMoveObject" is not executed. Only after reloading the page the object can be moved.

This bug exists since we updated to jQuery 3.1 and related modifications, e.g. new js-tree.

I tried to find a way to fix it, but don't really figure out how to do it :-)

**Comment by juergmessmer** [comment]
is duplicate of #246
